src/retasc/product_pages_api.py

```diff
@@ -0,0 +1,130 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Production Pages API
+"""
+
+from functools import cache
+
+from retasc import requests_session
+
+
+class ProductPagesApi:
+    """
+    Product Pages API Client
+    """
+
+    def __init__(self, api_url: str):
+        self.api_url = api_url
+        self.session = requests_session.requests_session()
+
+    def api_url_releases(self, id: str | None = None) -> str:
+        return f"{self.api_url.rstrip('/')}/releases/{id or ''}"
+
+    @cache
+    def releases_by_product_phase(
+        self,
+        product_shortname: str,
+        active_only: bool = True,
+        fields: str = "shortname,product_shortname,ga_date,phase_display",
+    ) -> list[dict]:
+        """
+        https://{pp_release_url}?product__shortname={product_shortname}&phase__lt=Unsupported&fields=a,b,c
+        The response have the following fields:shortname,product_shortname,ga_date,phase_display
+        phase__lt=Unsupported means "any supported phase" (these are less than Unsupported).
+        """
+
+        opt = {
+            "product__shortname": product_shortname,
+            "fields": fields,
+        }
+        if active_only is True:
+            opt.update({"phase__lt": "Unsupported"})
+        res = self.session.get(self.api_url_releases(), params=opt)
+        res.raise_for_status()
+        return res.json()
+
+    @cache
+    def release_schedules(
+        self,
+        release_short_name: str,
+        fields: str = "name,date_start,date_finish,flags",
+    ) -> list[dict]:
+        """
+        https://{pp_release_url}{release_short_name}/schedule-tasks?fields=a,b,c
+        The response have the following fields:name,,date_start,date_finish,flags
+        """
+
+        res = self.session.get(
+            f"{self.api_url_releases()}{release_short_name}/schedule-tasks",
```

Comment:
```suggestion
            f"{self.api_url_releases(release_short_name)}/schedule-tasks",
```

But better to create a helper method:

```python
    def api_url_release_schedule_tasks(self, id: str) -> str:
        return f"{self.api_url_releases(id)}/schedule-tasks",
```